packages/showcases/svelte/stories/components/selection-controls/VtmnTooltip/VtmnTooltip.csf.js

```diff
@@ -0,0 +1,27 @@
+export const argTypes = {
+  position: {
+    type: { name: 'string', required: false },
+    description: 'The tooltip position',
+    defaultValue: 'top',
+    control: {
+      type: 'radio',
+      options: [
+        'bottom-left',
+        'bottom',
+        'bottom-right',
+        'left',
+        'right',
+        'top-left',
+        'top',
+        'top-right',
+      ],
+    },
+  },
+
+  tooltip: {
+    type: { name: 'string', required: true },
+    description: 'The printed tooltip',
+    defaultValue: 'Tooltip text',
+    control: { type: 'text' },
+  },
+};
```

Comment:
Can you please use this existing file instead of yours? https://github.com/Decathlon/vitamin-web/blob/main/packages/showcases/core/csf/components/overlays/tooltip.csf.js
Thanks

Reply:
:( i did it first but on this one (for react) they use 'children'  and we don't want it on our controls